rebar.lock

```diff
@@ -1,2 +1,6 @@
-[{<<"jsx">>,{pkg,<<"jsx">>,<<"2.8.0">>},0},
+[{<<"eini">>,
```

Comment:
@fp may be do like here https://github.com/erlcloud/erlcloud/blob/master/rebar.lock#L1?